src/components/table/base-table/base-table.js

```diff
@@ -422,41 +462,79 @@ export default class BaseTable extends React.Component {
             </Cell>
           </div>
         )}
-      </CellMeasurer>
+      </ClassNames>
     );
   };
+
+  itemRenderer = renderParams => (
+    <CellMeasurer
+      cache={this.cellMeasurerCache}
+      columnIndex={renderParams.columnIndex}
+      key={renderParams.key}
+      parent={renderParams.parent}
+      rowIndex={renderParams.rowIndex}
+    >
+      {this.renderItemCell(renderParams)}
```

Comment:
Just to make the tests setup a bit more clean

Reply:
💯